maestro-ios-driver/src/main/kotlin/xcuitest/XCTestDriverClient.kt

```diff
@@ -225,34 +237,31 @@ class XCTestDriverClient(
     }
 
     private fun OkHttpClient.Builder.addRetryInterceptor() = addInterceptor(Interceptor { chain ->
-        var exception: IOException? = null
-        var response: Response? = null
-
-        repeat(3) {
-            try {
-                response?.close()
-                response = chain.proceed(chain.request())
-                if (response?.isSuccessful == true) {
-                    return@Interceptor response!!
-                }
-            } catch (e: IOException) {
-                installer.start()?.let {
-                    client = it
-                }
-                exception = e
-            }
-            Thread.sleep(300)
+        val response = try {
+             chain.proceed(chain.request())
+        } catch (ioException: IOException) {
+            val networkException = mapNetworkException(ioException)
+            networkErrorHandler.reinitializedXCTestClient()?.let { client = it }
```

Comment:
Might need to do this after installation

Reply:
Hm, not sure I follow - could you elaborate on this? What do we need to do after installation? Installation of the XCTest runner app?